seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java

```diff
@@ -73,6 +72,30 @@ protected final void prepare(E env, List<? extends Plugin<E>>... plugins) {
         }
     }
 
+    /**
+     * Execute close method defined in {@link org.apache.seatunnel.plugin.Plugin}
+     *
+     * @param plugins plugin list
+     */
+    @SafeVarargs
+    protected final void close(List<? extends Plugin<E>>... plugins) {
+        Optional<RuntimeException> exceptionHolder = Optional.empty();
+        for (List<? extends Plugin<E>> pluginList : plugins) {
+            for (Plugin<E> plugin : pluginList) {
+                try (Plugin<?> closed = plugin) {
+                    // ignore
+                } catch (Exception e) {
+                    exceptionHolder = Optional.of(new RuntimeException(String.format(
+                            "plugin %s close error", plugin.getClass().getName()), exceptionHolder.orElse(null)));
```

Comment:
If mutiple plugins close failedly, the subsequent  exception will cover the previous exception and we will lose the previouse exception info.
Maybe we can save all exception firstly (e.g. plugin -> exception) , continue executing next plugin's `close()` function, and throw the exception or return error message to notify the user which plugin close failedly.

Reply:
@Rianico Yes, has modified as you said, and all the plugin error message will be stored now. Please take a look.